src/MUIDataTable.js

```diff
@@ -1069,11 +1071,24 @@ class MUIDataTable extends React.Component {
             const selected = isRowSelectable ? isRowSelectable(displayData[i].dataIndex, prevSelectedRows) : true;
             selected && arr.push({ index: i, dataIndex: displayData[i].dataIndex });
             return arr;
-          }, []);
+           }, []);
 
           let newRows = [...prevState.selectedRows, ...selectedRows];
           let selectedMap = buildMap(newRows);
 
+          // if the select toolbar is disabled, the rules are a little different
+          if (this.options.disableToolbarSelect === true) {
+            if (selectedRowsLen > displayData.length) {
+               isDeselect = true;
```

Comment:
Looks like the merge eliminated the declaration change from `const` to `let` for `isDeselect`.

Reply:
Updated. Also, I've given you commit privileges to the branches I've PR-ed, just so you're not waiting on me if you see anything small like this.